src/pkg/scan/vuln/report.go

```diff
@@ -109,4 +109,41 @@ type VulnerabilityItem struct {
 	// The artifact digest which the vulnerability belonged
 	// e.g: sha256@ee1d00c5250b5a886b09be2d5f9506add35dfb557f1ef37a7e4b8f0138f32956
 	ArtifactDigest string `json:"artifact_digest"`
+	// The CVSS3 and CVSS2 based scores and attack vector for the vulnerability item
+	CVSSDetails CVSS `json:"preferred_cvss"`
+	// A separated list of CWE Ids associated with this vulnerability
+	// e.g. CWE-465,CWE-124
+	CWEIds []string `json:"cwe_ids"`
+	// A collection of vendor specific attributes for the vulnerability item
+	// with each attribute represented as a key-value pair.
+	// e.g. [{"key": "foo", "value":"bar"}]
+	VendorAttributes []VendorAttribute `json:"vendor_attributes"`
```

Comment:
```suggestion
	VendorAttributes map[string]interface{} `json:"vendor_attributes"`
```

Reply:
Converted to `map[string]interface{}` as discussed. This comment has been addressed